dashboard/app/views/devise/sessions/_login_lti_account_linking.haml

```diff
@@ -33,6 +33,9 @@
           = f.label :password
           = f.password_field :password, autofocus: email != '', required: true
 
+        = f.hidden_field :lti_provider, value: params[:lti_provider]
```

Comment:
Haha I added this same change to the disallow admin from linking accounts feature. Guess I can just rebase this in now :)